Submitted By:            Matt Burgess <matthew_at_linuxfromscratch_dot_org>
Date:                    2010-07-26
Initial Package Version: 2.11.2
Upstream Status:         Not submitted
Origin:                  Denis Onischenko
                         (http://www.mail-archive.com/help-make@gnu.org/msg08847.html)
Description:             Fixes an incompatibility with Make-3.82.

diff -Naur glibc-2.11.2.orig/manual/Makefile glibc-2.11.2/manual/Makefile
--- glibc-2.11.2.orig/manual/Makefile	2010-05-19 20:38:20.000000000 +0000
+++ glibc-2.11.2/manual/Makefile	2010-07-26 06:51:27.049899308 +0000
@@ -232,9 +232,13 @@
 .PHONY: stubs
 stubs: $(objpfx)stubs
 endif
-$(objpfx)stubs ../po/manual.pot $(objpfx)stamp%:
-	$(make-target-directory)
-	touch $@
+$(objpfx)stubs ../po/manual.pot:
+	$(make-target-directory)
+	touch $@
+
+$(objpfx)stamp%:
+	$(make-target-directory)
+	touch $@
 
 # Make the target directory if it doesn't exist, using the `mkinstalldirs'
 # script that does `mkdir -p' even if `mkdir' doesn't support that flag.
